package.json

```diff
@@ -3,10 +3,11 @@
   "description": "Access the Téléservice Identité Nationale de Santé with a js API",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "version": "3.1.0",
+  "version": "3.2.0",
   "scripts": {
-    "build": "npm run clean && tsc",
+    "build": "npm run clean:build && tsc",
     "clean": "tsc --build --clean",
+    "clean:build": "rimraf dist",
```

Comment:
no need to add dependencies, `rm -rf dist` is [enough](https://www.ashleysheridan.co.uk/blog/Why+You+Should+Not+Use+Rimraf)